spec/models/user_spec.rb

```diff
@@ -1,8 +1,13 @@
 require "rails_helper"
 
 describe User do
+  let(:user) { create(:user) }
+
+  it 'is not an admin when created' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.